### PR TITLE
Fix Missing collection association alias

### PR DESCRIPTION
**Pull request by jdeniau**
This PR follows #209 and fixes OneToMany associations

### CODE REVIEW

File: Doctrine/Orm/Filter/SearchFilter.php [hunk 1]
```diff
@@ -131,8 +131,8 @@ public function apply(ResourceInterface $resource, QueryBuilder $queryBuilder)
                 $values = array_map([$this, 'getFilterValueFromUrl'], $values);
 
                 $queryBuilder
-                    ->join(sprintf('o.%s', $property), $property)
-                    ->andWhere(sprintf('%1$s.id IN (:%1$s)', $property))
+                    ->join(sprintf('o.%s', $property), 'api_'.$property)
+                    ->andWhere(sprintf('%s.id IN (:%s)', 'api_'.$property, $property))
```
Comment (dunglas) [inline Doctrine/Orm/Filter/SearchFilter.php]: api_ can be directly in the sprintf pattern here (easier to read)

**Comment by sroze** [comment]
Thanks :+1:

**Comment by sroze** [comment]
Before merging (or not), we need to solve the discussion in #195.

**Comment by teohhanhui** [comment]
@sroze This does not depend on #195

**Comment by dunglas** [comment]
Thank you @jdeniau

**Pull request merged**